.github/workflows/docs.yml

```diff
@@ -21,4 +21,4 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: doc
-          args: --manifest-path openmls/Cargo.toml --message-format json
+          args: --message-format json
```

Comment:
This looks like we're building all docs now? We don't want to have docs published for the PoC crates.